Buildings/Fluid/HeatExchangers/WetCoilCounterFlow.mo

```diff
@@ -179,11 +184,5 @@ First implementation.
         preserveAspectRatio=true,
         extent={{-100,-100},{100,100}},
         grid={2,2},
-        initialScale=0.5), graphics={Text(
-          extent={{60,72},{84,58}},
-          lineColor={0,0,255},
-          textString="water-side"), Text(
-          extent={{50,-32},{90,-38}},
-          lineColor={0,0,255},
-          textString="air-side")}));
+        initialScale=0.5)));
```

Comment:
I removed the duplicated text (already present in the superclass `Fluid.HeatExchangers.DryCoilCounterFlow`).